quartz/build.gradle

```diff
@@ -88,3 +88,14 @@ processResources {
         expand([version: project.version, fullname: project.fullname, name: project.name])
     }
 }
+
+ext {
+    JDK_PATH = System.getProperty("jdk")
+}
+
+tasks.withType(Test).configureEach {
+    if ( JDK_PATH != null  ) {
+        println("External Java home '${JDK_PATH}' spefified for tests!")
```

Comment:
These changes are not necessary, also typo here. Changes to build.gradle should be reverted.